react/react1/week3/to-do/src/components/MyChildren.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+const MyChildren = (props) => {
```

Comment:
I supposed this component is made to just test out the `children` behaviour. It is good for testing and learning purposes, but other than that, it is kind of redundant in this app. 🙏🏽

Reply:
Yes,in home work they asked to make a child component,may be I understood something wrongly and yes I made this to understand how children props work in react,thank you for your time 🙏🏾.

Reply:
Thank you very much for your review and feed back,ai require this much for further improvement 🙏🏾.